keyboards/treadstone48/rev1/info.json

```diff
@@ -0,0 +1,257 @@
+{
+  "keyboard_name": "Treadstone48",
+  "url": "https://github.com/marksard/Keyboards",
+  "maintainer": "marksard",
+  "width": 14.75,
+  "height": 4,
+  "layouts": {
+    "LAYOUT": {
+      "layout": [
+        {
```

Comment:
You have 47 keys designated [here](https://github.com/qmk/qmk_firmware/blob/f2c1aa69d18b70de6a4e23e5441d8096a61bc519/keyboards/treadstone48/rev1/info.json#L10-L253), but there needs to be 48, because `LAYOUT_base` designates 48 keys.

https://github.com/qmk/qmk_firmware/blob/f2c1aa69d18b70de6a4e23e5441d8096a61bc519/keyboards/treadstone48/rev1/rev1.h#L55-L60

They also need to be assigned in the same order as they are in `keyboards/treadstone48/rev1/rev1.h` Lines 55 through 60.

Reply:
got it thanks:)